cfpb/capital-framework
# cf-notifications: Target only notification icon

Notifications that contain an external link are getting an extra SVG icon in the notification that is being absolutely positioned. See, for instance, https://www.consumerfinance.gov/about-us/blog/fall-2015-rulemaking-agenda/

<img width="795" alt="Screen Shot 2019-05-17 at 11 31 00 AM" src="https://user-images.githubusercontent.com/704760/57938960-471d2d00-7897-11e9-8191-c0659850e1e0.png">

This PR fixes that by targeting the SVG directly inside of the notification container, which is the notification's icon only.

## Changes

- Target notification icon directly for positioning.

## Testing

1. Pull branch down and run `cd docs && gulp && bundle exec jekyll serve`
2. Visit http://localhost:4000/components/cf-notifications/ and check the notification with an external link.

## Screenshots

Before:
<img width="896" alt="Screen Shot 2019-05-17 at 11 23 57 AM" src="https://user-images.githubusercontent.com/704760/57939095-877cab00-7897-11e9-9de1-530470b9cf59.png">

After:
<img width="894" alt="Screen Shot 2019-05-17 at 11 27 07 AM" src="https://user-images.githubusercontent.com/704760/57939106-8d728c00-7897-11e9-95a5-08b38a1ddc7c.png">
